### PR TITLE
Release 0.0.2

### PR DESCRIPTION
**Pull request by PravinPK**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/cordova-acpgriffon",
-  "version": "0.0.1",
```
Comment (ravjain-adb) [inline package.json:3]: lets move to 1.0.0 ?

Comment (PravinPK) [reply package.json:3]: this is just an update to Griffon SDK, we can keep 0.0.2

I will do 1.0.0 when we rename as Assurance SDK

File: plugin.xml [hunk 2]
```diff
@@ -48,7 +48,7 @@
                 <source url="https://github.com/CocoaPods/Specs.git" />
             </config>
             <pods use-frameworks="true">
-                <pod name="ACPGriffon" spec=">= 1.1.0" />
```
Comment (ravjain-adb) [inline plugin.xml:51]: In this file line 3 u want to change that version as well right

Comment (PravinPK) [reply plugin.xml:51]: Good catch, I didn't see that

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,2 +1,5 @@
+### 0.0.1 (5 Nov, 2020)
```
Comment (sbenedicadb) [inline CHANGELOG.md]: 0.0.2?

**Pull request merged**